### PR TITLE
Replaced the prints through warnings

### PR DESCRIPTION
**Pull request by Eposs74**
# Related Issues
#168 
#191 
# Notes
Replaced all prints in the fields through warnings, wrote tests for all of them

### CODE REVIEW

**Comment by Eposs74** [comment]
I just have some questions, how to write tests for the warnings.

File: tests/test_fields/test_CurrentLine.py [hunk 1]
```diff
@@ -61,9 +61,11 @@ def test_Bfield_singularity():
     origin2 = vertices[1]
     middle = ((vertices[0]) + (vertices[1])) / 2
 
+
```
Comment (lucasgcb) [inline tests/test_fields/test_CurrentLine.py]: OH NO, A BLANK LINE

Comment (Eposs74) [reply tests/test_fields/test_CurrentLine.py]: I will do it INSTANDLY

Comment (lucasgcb) [reply tests/test_fields/test_CurrentLine.py]: :+1:

**Pull request merged**